### ISSUE TITLE
THIS IS WAR!!!!

### ISSUE THREAD

**Issue by Hardwaregore** [Jmaxdnix1/Jmaxdnix1.github.io#1]
@AdoptMeTradingValues @Jmaxdnix1

**Comment by Jmaxdnix1** [comment]
@Hardwaregore untill you can beat https://captainchicky.github.io/Idiot!/Converted/Idiot!.html me and a friend made that then you are the true IDIOT

**Comment by Hardwaregore** [comment]
no, I don’t think I will. I already achieved my main goal which was to get all of my teachers to run that. good luck fighting this unnecessary battle MR. @AdoptMeTradingValues

[#]~$Hardwaregore

**Comment by Jmaxdnix1** [comment]
I've achieved my goal of stealing your info. and selling it to "Josh" for $72

**Comment by Hardwaregore** [comment]
@AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues @AdoptMeTradingValues 

# (sarcasm) OMG, YOU DID!!!! GOOD JOB MR 'hacker'!!!!!!!! I TOTALLY REALLY CARE ABOUT THIS!!!!!!!

**Comment by Jmaxdnix1** [comment]
you're a retard your dad left you your mom thought of leaving you but couldn't because she couldn't leave a retarded child such as yourself to die. hope you enjoy being stalked by "Josh" and see you never.
